sql/core/src/main/scala/org/apache/spark/sql/execution/joins/MergeAsOfJoinExec.scala

```diff
@@ -124,7 +138,7 @@ case class MergeAsOfJoinExec(
             GroupedIterator(leftIter, Seq(leftBy), left.output)
           if (leftGroupedIterator.hasNext) {
             var currLeft = leftGroupedIterator.next()
-            match_tolerance(currLeft, currRight, duration, resultProj)
+            match_tolerance(currLeft, currRight, resultProj)
```

Comment:
match_tolerance function no longer takes a tolerance args, which means this function is no longer doing what its name suggests. What does this function do now?